sklearn/preprocessing/_encoders.py

```diff
@@ -210,6 +223,10 @@ class OneHotEncoder(_BaseEncoder):
         (in order of the features in X and corresponding with the output
         of ``transform``).
 
+    drop_ : array
```

Comment:
shape?

Reply:
I think the implementation can be simplified with little harm to the API, if we make it `drop_idx_` such that `categories_[i][drop_idx_[i]]` will be dropped for all `i`. Then fitting `drop='first'` just involves `drop_idx_ = np.zeros(len(self.categories_))`. At transform time the masked column indices are `drop_idx_ + np.cumsum([0] + [len(c) for c in categories_[:-1]])`.

Reply:
This is a good idea, I think I'll update the implementation to reflect this.